static/bidder-params/yieldlab.json

```diff
@@ -27,7 +23,6 @@
   },
   "required": [
     "adslotId",
-    "supplyId",
-    "adSize"
```

Comment:
Can you please update the docs as well [here](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/yieldlab.md) to remove the `adSize` parameter? Unless you still accept this parameter on the PBJS side

Reply:
Of course thank you, https://github.com/prebid/prebid.github.io/pull/3992